core/src/modules/optionHelper.ts

```diff
@@ -324,4 +324,26 @@ export namespace OptionHelper {
 
     return options;
   }
+
+  async function getDefaultOptionValues(
+    instance: Source | Destination | Schedule | Property | App
```

Comment:
At the moment only App, Source and Destination are implemented as stated in the story. Should I look into these other types?

Reply:
From standup: not for now